src/Microsoft.DocAsCode.Common/JsonUtility.cs

```diff
@@ -20,24 +21,28 @@ public static class JsonUtility
                     return jsonSerializer;
                 });
 
-        public static void Serialize(TextWriter writer, object graph, Formatting formatting = Formatting.None, JsonSerializer serializer = null)
+        public static void Serialize(TextWriter writer, object graph, Formatting formatting = Formatting.None, JsonSerializer serializer = null, object additional = null)
         {
             var localSerializer = serializer ?? DefaultSerializer.Value;
             localSerializer.Formatting = formatting;
+            if (additional != null)
+            {
+                localSerializer.Context = new StreamingContext(StreamingContextStates.Other, additional);
+            }
```

Comment:
bad status:
Call this method twice, first time with some additional, second time without additional.